### DISCUSSION TITLE
Laravel Livewire Sortable

### DISCUSSION THREAD

**Discussion by adetola-dotbat** [livewire/livewire#5633] [1. Help]
Drag and drop sorting is not updating in my database
![image](https://user-images.githubusercontent.com/105286553/228277226-699d15a0-5dd4-4fb0-b4f5-c12f995e3b09.png)
![image](https://user-images.githubusercontent.com/105286553/228277670-daf1e9ae-6655-4be3-91f0-7ca4d48f9d75.png)
![image](https://user-images.githubusercontent.com/105286553/228277914-95652226-d572-4a01-8aad-e925d04ceb1f.png)
Please is there anything I'm doing wrong

**Comment by joshhanley** [comment]
@adetola-dotbat please don't paste screenshots of code as it makes it hard for people to help you.
Instead paste code in code blocks.
When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Looking at code though, it all seems ok. Can you open devtools and go to the network tab, and the drag and drop something and see if a network request is being fired and confirm your `priority()` method is actually being called?
